test/stdlib/freestanding-check-size.swift

```diff
@@ -0,0 +1,27 @@
+// Check that we can LTO-optimize the freestanding/minimal stdlib against client usage of APIs, and that we don't produce unexpectedly large final binary.
+
+// Important (!): This test is in test/stdlib/ to make sure it is actually run on the minimal/freestanding CI job, which
+// filters the set of tests to test/stdlib/ only (see build-preset.ini).
+
+// REQUIRES: freestanding
+// REQUIRES: executable_test
+// UNSUPPORTED: objc_interop
```

Comment:
Is it not sufficient to just do `-disable-objc-interop` or is there something within the runtime that you are worried will persist?

Reply:
This is a leftover... I don't think Obj-C interop is relevant here at all. Will remove.